src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
Also as you see in the others issue it would be very handy if it could find dead shared code across assemblies as ILLink cannot.